figures/serializers.py

```diff
@@ -562,9 +562,9 @@ class LearnerCourseDetailsSerializer(serializers.ModelSerializer):
             }
     """
 
-    course_name = serializers.CharField(source='course_overview.display_name')
```

Comment:
The change from `course_overview` to `course` is because of the change in CourseEnrollment in Hawthorn